taca/backup/backup.py

```diff
@@ -257,39 +276,61 @@ def _clean_tmp_files(self, files):
 
     def _log_pdc_statusdb(self, run):
         """Log the time stamp in statusDB if a file is succussfully sent to PDC."""
-        try:
-            run_vals = run.split("_")
-            if len(run_vals[0]) == 8:
-                run_date = run_vals[0][2:]
-            else:
-                run_date = run_vals[0]
-            run_fc = f"{run_date}_{run_vals[-1]}"
-            couch_connection = statusdb.StatusdbSession(self.couch_info).connection
-            db = couch_connection[self.couch_info["db"]]
-            fc_names = {e.key: e.id for e in db.view("names/name", reduce=False)}
-            d_id = fc_names[run_fc]
-            doc = db.get(d_id)
-            doc["pdc_archived"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-            db.save(doc)
-            logger.info(
-                f'Logged "pdc_archived" timestamp for fc {run} in statusdb doc "{d_id}"'
-            )
-        except:
-            logger.warn(f'Not able to log "pdc_archived" timestamp for run {run}')
+        if "AV" not in run:
+            try:
+                run_vals = run.split("_")
+                if len(run_vals[0]) == 8:
+                    run_date = run_vals[0][2:]
+                else:
+                    run_date = run_vals[0]
+                run_fc = f"{run_date}_{run_vals[-1]}"
+                couch_connection = statusdb.StatusdbSession(self.couch_info).connection
+                db = couch_connection[self.couch_info["db"]]
+                fc_names = {e.key: e.id for e in db.view("names/name", reduce=False)}
+                d_id = fc_names[run_fc]
+                doc = db.get(d_id)
+                doc["pdc_archived"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+                db.save(doc)
+                logger.info(
+                    f'Logged "pdc_archived" timestamp for fc {run} in statusdb doc "{d_id}"'
+                )
+            except:
+                logger.warning(
+                    f'Not able to log "pdc_archived" timestamp for run {run}'
+                )
+        else:
+            try:
+                element_db_connection = statusdb.ElementRunsConnection(
+                    self.CONFIG.get("statusdb", {}), dbname="element_runs"
+                )
+                run_doc = element_db_connection.get_db_entry(run)
+                run_doc["pdc_archived"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+                element_db_connection.upload_to_statusdb(run_doc)
```

Comment:
This does not work the way I thought, I need to rewrite it

Reply:
Let me know when ready for re-review :)

Reply:
I found one issue and fixed it, but it's still not working and I need some help. Basically I want to fetch the doc from statusdb, add a timestamp to it, then upload it again. But the upload doesn't work, it can't find the key "name" in the document for some reason.

Reply:
Now it works! Thanks @aanil 🙏